test/edu/csus/ecs/pc2/imports/ccs/ContestSnakeYAMLLoaderTest.java

```diff
@@ -1286,7 +1287,254 @@ public void testValidatorKeys() throws Exception {
 
     }
 
-    // SOMEDAY get this JUnit working
```

Comment:
Is this a SOMEDAY or a TODO ?

Reply:
First, aren't ALL "TODOs" effectively "someday"?   If it's something "to be done", it's by definition NOT currently done, so it "needs to be done", and that could be later today, tomorrow, next week, next month, next year... at some point in the future -- i.e. "someday".
Second, we do not have, have never had, any formal specification of what anything like "TODO" or "SOMEDAY" means in a comment.  Not saying we wouldn't benefit by having such; just saying we do not have any such thing and never have had any such thing.  So asking "Is this a SOMEDAY or a TODO" is pretty much a moot question since there's nothing anywhere that defines what either of those formally means.
Third, **I did not put this comment in the code**; it was there when I started working on this PR. 

I don't think this is an important issue, at least not with regard to this PR.  If you disagree, feel free to file a new Issue on it.